target/linux/lantiq/image/Makefile

```diff
@@ -549,9 +581,10 @@ ifeq ($(SUBTARGET),xrx200)
 
 define Device/zyxel_p-2812hnu-f1
   $(Device/NAND)
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := P-2812HNU-F1
```

Comment:
Is F1/F3 a variant or part of the model name here?

Reply:
The technical specs document is for ..Fx, so I assume it's a revision.